core/access.php

```diff
@@ -0,0 +1,27 @@
+<?php
+defined('CMSPATH') or die; // prevent unauthorized access
+
+class Access {
+    public static function can_access($page_groups=[], $user_groups=[]) {
+        //prep and parse the inputs
+        if (sizeof($user_groups) == 0) {
```

Comment:
Minor style/performance thing - but for testing if an array is empty I'd encourage either 

```php
if (!$user_groups)
```
or 
```php
if (empty($user_groups))
```

I'd probably use the former myself, but could see benefits for clarity in using the second. Both are clearer, less error prone when implementing (imho), and potentially more performant if the array is large.

Reply:
https://github.com/HoltBosse/Alba/pull/143/commits/4b475a6b3b376304a540224f53a13275c4fb6ec7 resolved